guides/hack/03-expressions-and-operators/01-introduction.md

```diff
@@ -1,29 +1,46 @@
-* [Comparisons, `>` etc](comparisons.md)
-* [Decrement, `--`](incrementing-and-decrementing.md)
-* [echo intrinsic function](echo.md)
-* [Equality operators](equality.md)
-* [Error control, `@`](error-control.md)
-* [exit intrinsic function](exit.md)
-* [Increment, `++`](incrementing-and-decrementing.md)
-* [invariant intrinsic function](invariant.md)
-* [list intrinsic function](list.md)
-* [Logical operators, `&&`, `||`, `!`](logical-operators.md)
-* [Member selection, `->`, `?->`](member-selection.md)
-* [new](new.md)
-* [Operator precedence](operator-precedence.md)
-* [Pipe, `|>`](pipe.md)
-* [Scope resolution, `::`](scope-resolution.md)
-* [Spaceship, `<=>`](equality.md)
-* [String concatenation, `.`](string-concatenation.md)
-* [Subscripting, `[]`](subscript.md)
-* [Ternary, `?` `:`](ternary.md)
-* [Type Assertions `is`, `as`, `?as`](type-assertions.md)
-* [Yield](yield.md)
+When combined, operators evaluate according to their associativity. For more information, see [Operator Precedence](/hack/expressions-and-operators/operator-precedence).
+
+## Assignment Operators
+* [Assignment](/hack/expressions-and-operators/assignment) (`=`, `+=`, and more)
+* [Coalescing Assignment](/hack/expressions-and-operators/coalesce#coalescing-assignment-operator) (`??=`)
+
+## Comparison Operators
+* [Comparison](/hack/expressions-and-operators/comparisons) (`>`, `>=`,`<`,`<=`)
+* [Equality Comparison](/hack/expressions-and-operators/equality) (`==`, `!=`, `===`,`!==`)
+* [Logical Comparison](/hack/expressions-and-operators/logical-operators) (`&&`, `||`, `!`)
+* [Spaceship Comparator](/hack/expressions-and-operators/equality#the-spaceship-operator) (`<=>`)
+
+## Arithmetic Operators
+* [Arithmetic](/hack/expressions-and-operators/arithmetic) (`+`, `-`, `*`, `/`, and more)
+* [Increment](/hack/expressions-and-operators/incrementing-and-decrementing) (`++`)
+* [Decrement](/hack/expressions-and-operators/incrementing-and-decrementing) (`--`)
+
+## Bitwise and Bit Shift Operators
+* [Bitwise](/hack/expressions-and-operators/bitwise-operators) (`&`, `|`, `^`, `<<`, `>>`, `~`)
+
+## Class and Member Operators
+* [Create an Object](/hack/expressions-and-operators/new) (`new`)
+* [Access Instance Members](/hack/expressions-and-operators/member-selection) (`->`, `?->`)
+* [Access Static Members / Enum Properties](/hack/expressions-and-operators/scope-resolution) (`::`)
+* [Access XHP Attributes](/hack/expressions-and-operators/XHP-attribute-selection) (`->:`)
+
+## Built-in Type Operators
+* [Cast/Convert Types](/hack/expressions-and-operators/casting) (`(int)`, `(string)`, and more)
+* [Check Types](/hack/expressions-and-operators/type-assertions#checking-types-with-is) (`is`)
+* [Enforce Types](/hack/expressions-and-operators/type-assertions#enforcing-types-with-as-and-as) (`as`, `?as`)
+* [Concatenate a String](/hack/expressions-and-operators/string-concatenation) (`.`)
+* [Index Hack Arrays and Strings](/hack/expressions-and-operators/subscript) (`[]`)
```

Comment:
How about 'index into hack arrays and strings'? Otherwise this sentence is a little hard to parse, because you have to work out if you're talking about index as a noun or verb.